.github/workflows/pr-bot.yml

```diff
@@ -16,8 +16,12 @@ jobs:
         uses: ./ # uses the local action defined by action.yml in the root of the repository
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          GITHUB_TOKEN: ${{ secrets.PR_BOT_TOKEN }}
 
       - name: Invoke action
         uses: marqeta/pr-bot-cli@main
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
```

Comment:
No need to have both, so just replace the `env` GITHUB_TOKEN? That way no other changes to code.

Reply:
I think we should keep `with` and remove `env`. That way our customers know that they need to give a github token as input. 

The reason why I kept `with` here is the remote action is still using old code, if I remove `with`, remote action will fail, and thus fail required status check.